README.md

```diff
@@ -18,7 +18,7 @@ This library uses the [Application Default Credentials](https://cloud.google.com
 connection to the Cloud SQL server. For more details, see the previously
 mentioned link.
 
-To activate credentials locally, use the following `gcloud` command:
+To activate credentials locally ensure the Google Cloud SDK is installed on your machine. For manual installation see [Installing Cloud SDK](https://cloud.google.com/sdk/docs/install). macOS users can use `brew install --cask google-cloud-sdk` from the terminal. Once installed, use the following `gcloud` command:
```

Comment:
The page linked has macOS instructions. It seems best to just let the user use those. 
```suggestion
To activate credentials locally ensure the Google Cloud SDK is installed on your machine. For manual installation see [Installing Cloud SDK](https://cloud.google.com/sdk/docs/install). Once installed, use the following `gcloud` command:
```

Reply:
Agreed